dbt-labs/dbt-spark
# Add http error identification for hostname and credentials

Internal ticket: https://jira.cloudera.com/browse/DBT-326

The error printed seems to be generic, so adding
code to identity the type request failure from Livy server.

Test plan:
    * Move into a dbt spark livy project
    * Install dbt-spark-livy from this git repo
    * Add profile details in your home dbt file like from thi page
    https://community.cloudera.com/t5/Innovation-Blog/Getting-started-with-dbt-spark-livy-adapter/ba-p/347360
    * Update with wrong hostname or wrong user credentials
    * run > dbt debug, and you should see the type of error in the console
    * Add below snowplow configs to your .env if you face any snowplow error
      SNOWPLOW_ENDPOINT=https://dcevents.cldrteam.datacoral.io/cldrteam/apievents
      SNOWPLOW_TIMEOUT=10
      SNOWPLOW_API_KEY=P8TkX3zUwtOIxJkoKngn3Rf9j3jyro6Bfnd50AEV
      SNOWPLOW_ENNV=prod

Signed-off-by: Sanjeev kumar N <sanjeevitcit@gmail.com>
